src/configuration.js

```diff
@@ -63,5 +63,6 @@ export const navButtonStyles = {
   bottom: '0',
   width: '100%',
   textAlign: 'center',
-  backgroundColor: '#fff',
+  backgroundColor: '#0369a1',
```

Comment:
Should this be moved to tailwind css? (happy to help on slack)

Reply:
Yes, there are actually several little things that need to be switched from css to tailwind. I focused on broad strokes for this round and will dig in to the old stuff next.